### PR TITLE
 e2e: create a test for core harvest with RT

### PR DESCRIPTION
**Pull request by szymonlopaciuk**
## Checklist:

- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [ ] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/e2e/scenarios/default/RTService/authenticate.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+request:
+  headers:
+    Host: [inspirevm13.cern.ch]
```
Comment (david-caro) [inline tests/e2e/scenarios/default/RTService/authenticate.yaml]: Can you change this to be rt.inspirehep.net?

Comment (david-caro) [reply tests/e2e/scenarios/default/RTService/authenticate.yaml]: (and all the related places, like the url below)

**Comment by ammirate** [comment]
ci test please

**Comment by ammirate** [comment]
ci test please

**Comment by ammirate** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci merge please

**Pull request merged**